src/udev/net/link-config-gperf.gperf

```diff
@@ -37,6 +37,7 @@ Link.Duplex,                     config_parse_duplex,        0,
 Link.AutoNegotiation,            config_parse_tristate,      0,                             offsetof(link_config, autonegotiation)
 Link.WakeOnLan,                  config_parse_wol,           0,                             offsetof(link_config, wol)
 Link.Port,                       config_parse_port,          0,                             offsetof(link_config, port)
+Link.Advertise,                  config_parse_advertising,   0,                             offsetof(link_config, advertising[0])
```

Comment:
You still have "advertising" left in various parts of the code. Please change that to "advertise" where appropriate.